build_tools/ci/cpu_comparison/run.py

```diff
@@ -936,6 +1040,79 @@ def aie_vs_baseline(
             raise RuntimeError("Test failed, exiting.")
 
 
+def benchmark_aie(
+    config,
+    aie_compilation_flags,
+    test_file,
+    use_ukernel,
+    tile_pipeline,
+    lower_to_aie_pipeline,
+    function_name,
```

Comment:
This is needed. Without it you it can't find the function: `Failed to match any benchmarks against regex: .`. I would have expected it to work without as well.